roles/setup_hammerdbserver/tasks/install_hammerdb.yml

```diff
@@ -7,3 +7,22 @@
 - name: Install HammerDB
   raw: tar -xvf HammerDB-{{ hammerdb_version }}-Linux.tar.gz
   become: no
+
+# We need this Azure Database hack until this is resolved:
+# https://github.com/TPC-Council/HammerDB/issues/163
+
+- name: Create HammerDB patch for Azure Database
+  template:
+    dest: hammerdb-{{ hammerdb_version }}-azure-db.patch
+    src: hammerdb-{{ hammerdb_version }}-azure-db.patch.template
+    mode: 0644
+  become: no
+  when: azure_db_hackery is defined
```

Comment:
I'd suggest to check if `azure_db_hackery` is `True`

Reply:
Done,